builder/converter/templates/categorized_view_template.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<org.jenkinsci.plugins.categorizedview.CategorizedJobsView plugin="categorized-view@1.9">
+    <name></name>
+    <filterExecutors>false</filterExecutors>
+    <filterQueue>false</filterQueue>
+    <properties class="hudson.model.View$PropertyList"/>
+    <jobNames>
+        <comparator class="hudson.util.CaseInsensitiveComparator"/>
+    </jobNames>
+    <jobFilters/>
+    <columns>
+        <hudson.views.StatusColumn/>
+        <hudson.views.WeatherColumn/>
+        <org.jenkinsci.plugins.categorizedview.IndentedJobColumn/>
+        <hudson.views.LastSuccessColumn/>
+        <hudson.views.LastFailureColumn/>
+        <hudson.views.LastDurationColumn/>
+        <hudson.views.BuildButtonColumn/>
+    </columns>
+    <includeRegex></includeRegex>
+    <recurse>true</recurse>
+    <groupingRules/>
```

Comment:
I checked in a test environment by creating a categorized view on Jenkins web UI with one cetegorizationCriteria : these elements are defined as top level elements.

Here is the exported XML for the view I mentioned :

`<?xml version="1.0" encoding="UTF-8"?>
<org.jenkinsci.plugins.categorizedview.CategorizedJobsView plugin="categorized-view@1.10">
  <name>test-view</name>
  <filterExecutors>false</filterExecutors>
  <filterQueue>false</filterQueue>
  <properties class="hudson.model.View$PropertyList"/>
  <jobNames>
    <comparator class="hudson.util.CaseInsensitiveComparator"/>
  </jobNames>
  <jobFilters/>
  <columns>
    <hudson.views.StatusColumn/>
    <hudson.views.WeatherColumn/>
    <org.jenkinsci.plugins.categorizedview.IndentedJobColumn/>
    <hudson.views.LastSuccessColumn/>
    <hudson.views.LastFailureColumn/>
    <hudson.views.LastDurationColumn/>
    <hudson.views.BuildButtonColumn/>
  </columns>
  <recurse>true</recurse>
  <groupingRules/>
  <regexToIgnoreOnColorComputing></regexToIgnoreOnColorComputing>
  <categorizationCriteria>
    <org.jenkinsci.plugins.categorizedview.GroupingRule>
      <groupRegex>job-.*</groupRegex>
      <namingRule>All job with prefix &quot;job-&quot;</namingRule>
      <useDisplayName>false</useDisplayName>
    </org.jenkinsci.plugins.categorizedview.GroupingRule>
  </categorizationCriteria>
</org.jenkinsci.plugins.categorizedview.CategorizedJobsView>⏎`